### PR TITLE
Add MyIP extension

### PR DESCRIPTION
**Pull request by kkange**
## Description
 
get my public ip and see the details and torrent download history of ip.

## Type of change

- New extension

## Screencast
스크린샷 2021-12-14 03 16 43
스크린샷 2021-12-14 03 27 46
스크린샷 2021-12-14 03 18 51

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/myip/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "title": "MyIP",
+  "description": "My IP information",
+  "icon": "icon.png",
+  "author": "Kang",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show IP Address",
+      "subtitle": "Internet",
```
Comment (pernielsentikaer) [inline extensions/myip/package.json:13]:
```suggestion
```
Could you remove this one, it seems redundant to me

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kkange 👋 

Only a single correction, you're on fire 🔥 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by kkange** [comment]
hi @pernielsentikaer

Thank you for the feedback.

I reflected that.

**Comment by kkange** [comment]
I added these.

- Remove unused libraries
- Apply context-sensitive navigation movements after copying values ​​or opening a browser

**Review by pernielsentikaer — APPROVED** [review]
Hi @kkange 🔥 

Thanks for the update, I'm going to accept it and then will my colleague do a final review before it's finally merged

**Review by mattisssa — APPROVED** [review]
Wow, this is the good one 👍 
Let's bring it to the Store

**Comment by jankerli** [comment]
@kkange can add a feature to get the Local network ip？

**Comment by kkange** [comment]
> @kkange can add a feature to get the Local network ip？

If you want, I'll add it.

**Comment by jankerli** [comment]
cool~ thanks

**Comment by jankerli** [comment]
> > @kkange can add a feature to get the Local network ip？
> 
> If you want, I'll add it.
> 
> I did PR.

cool~ thanks

**Pull request merged**